README.md

```diff
@@ -1198,6 +1198,28 @@ params do
 end
 ```
 
+You can also create custom validation that use request to validate the attribute. For example if you want to have parameters that are available to only admins, you can do this
+```ruby
```

Comment:
Add a space above. Change this to a full phrase, eg " admins, you can do the following."